README.md

```diff
@@ -68,9 +76,35 @@ To compile the contracts:
 ./go contracts:compile
 ```
 
-## Testing
+---
+### Run
+To deploy instances of the contracts for local development without prior knowledge Ganache and Truffle, run the following command:
+```shell
+./go contracts:run
+```
+
+This command starts up Ganache on a random port and migrates all contracts to the Ganache instance. The Ganache instance will remain running the the background, even though the command prompt is available.
```

Comment:
"running the the background" should be "running in the background"

Reply:
Will change.